CODE_STYLE.md

```diff
@@ -62,11 +62,11 @@ Implementation
 
 * All instructions should be built as part of both the RV32 and RV64 models so
   as to provide a path to supporting mutable MXL/SXL/UXL; if necessary,
-  constructs like `assert(sizeof(xlen) == 32)` at the start of the body can be
+  constructs like `xlen == 32)` at the start of the body can be
```

Comment:
```suggestion
  constructs like `assert(xlen == 32)` at the start of the body can be
```

Not sure what happened here...

Reply:
Oops. Not sure how that happened. Fixed.